### ISSUE TITLE
Incompatible with React v15.4.0

### ISSUE THREAD

**Issue by erikras** [nikgraf/belle#330]
As of React `v15.4.0`, [you can no longer import from `react/lib/*`](https://facebook.github.io/react/blog/2016/11/16/react-v15.4.0.html) like this library is doing in [`injectStyles.js`](https://github.com/nikgraf/belle/blob/5004cf108f68406784c2ff22331e92e2da1e0ac4/src/utils/inject-style.js#L2). Results in:

```
Error: Cannot find module 'react/lib/CSSPropertyOperations'
```

**Comment by cezarneaga** [comment]
@nikgraf was looking for maintainers a couple of weeks ago. not sure he still has time for belle :)
a bit busy this week but will try something this weekend maybe

**Comment by ig43** [comment]
Yup - I've just run into exactly the same issue, after having done npm update. I'm using Webpack, so will see if I can set up an alias to point to the correct directory.

**Comment by nikgraf** [comment]
I will update this over the next couple days … sorry for the delay

**Comment by ig43** [comment]
Hi Nik,



Many thanks for getting in touch with an update on status.

From my point of view, I have temporarily worked around it, simply by creating an alias in webpack.config.js. So it’s only a minor issue.



I must say that you have done a fantastic job with Belle. I evaluated several UI libraries, such as MaterialUI etc., but Belle for me was the best and well documented.

So I owe you a big thank you!!



Kind regards,

Ian



Ian Gillingham

Senior Electronic Systems Engineer

Diamond Light Source

DH G.01

Didcot

Oxfordshire

UK

OX11 0DE



Tel: +44 (0)1235 778129

**Comment by nikgraf** [comment]
@erikras @cezarneaga @ig43 just published 3.0.0 https://www.npmjs.com/package/belle which is compatible with React 15.4.0 🙌

Thanks for your patience!

**Comment by erikras** [comment]
Finally got around to upgrading. ❤️ @nikgraf!!
